src/index.html

```diff
@@ -1,17 +1,169 @@
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      id="our-form"
+    >
+      <fieldset class="form-box">
+        <legend>Personal information</legend>
+        <div class="form-filed">
+          <label for="surname">Surname:
+            <input type="text" 
+            name="surname" 
+            id="surname"
+            autocomplete="off" 
+            required
+            >
+          </label> 
+        </div>
+        <div class="form-filed">
+          <label for="name"> Name:</label>
+          <input type="text" 
+          name="name" 
+          id="name"
+          autocomplete="off"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="age">How old are You?</label>
+          <input type="number"
+            name="age"
+            id="age"
+            pattern="\d+"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="date-birth">Full date of birth:</label>
+          <input type="date"
+            name="date-birth"
+            id="date-birth"
+            placeholder="date"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="accept">I accept the term of the agreement</label>
+          <input type="checkbox"
+            name="accept-agreement"
+            id="accept"
+            value="yes"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+        <div class="form-filed">
+          <label for="email">E-mail: </label>
+          <input type="email"
+            placeholder="email@example.com"
+            name="email"
+            id="email"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="password">Password: </label>
+          <input type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-filed">
+          <label for="cats"
+            >Do you love cats?
+            <input type="radio" 
+            name="cats"  
+            value="Yes"
+            > Yes
+            <input type="radio"
+             name="cats" 
+             value="No"
+            > No
+          </label>
```

Comment:
Fix formating.
Also you need to add lables to each radio button so when label is clicked the radio button is selected